### PR TITLE
I come to pulish your study. ;-)

### PR DESCRIPTION
**Pull request by ZhaoCake**
我就是太无聊了来看看你。

### CODE REVIEW

**Comment by RosalindJackson** [comment]
所以这是在干什么

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Some Questions
+
+1. May I know about the differences between `test1` and `test2` ,Yang?
```
Comment (RosalindJackson) [inline README.md:3]: there are no diffierece, just test it to see if i can upload some codes

Comment (ZhaoCake) [reply README.md:3]: Well, sounds not bad. May you merge this pull request to your repo? :-)

**Comment by RosalindJackson** [comment]
i do something, is it right?

**Comment by ZhaoCake** [comment]
> i do something, is it right?

Sure. You can also pull request to my repo.

**Pull request merged**